src/utils/network.py

```diff
@@ -1,11 +1,14 @@
-"""Buidling, visualizing, and analyzing networks"""
+"""Buidling, visualizing, and analyzing networks (micro-level)"""
```

Comment:
looks like there is macro stuff in here too

Reply:
Oops, will fix this typo in next commit!